modules/MinimumSpanningTree/src/main/java/com/carlschroedl/gephi/plugin/minimumspanningtree/GraphTopologyEquals.java

```diff
@@ -0,0 +1,88 @@
+package com.carlschroedl.gephi.plugin.minimumspanningtree;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import org.gephi.graph.api.Edge;
+import org.gephi.graph.api.Graph;
+import org.gephi.graph.api.Node;
+import org.gephi.graph.impl.utils.MapDeepEquals;
+import org.openide.util.Pair;
+
+/**
+ * This class determines whether two graphs have the same nodes and edges. Nodes
+ * from different graphs are considered equal if and only if their node ids are 
+ * equal. Edges from different graphs are considered equal if and only if their 
+ * source node ids, target node ids, and weights are equal. Two graphs are equal
+ * if and only if all of their nodes and edges are equal, and they have the same
+ * number of edges and nodes.
+ * All other node and edge attributes are ignored in this comparison.
+  */
+public class GraphTopologyEquals {
```

Comment:
I don't think there are builtin methods for this, but this is a small class. Great that you have tests!